papers/views.py

```diff
@@ -85,7 +90,7 @@ def fetch_on_orcid_login(sender, **kwargs):
     else:
         r.fetch_everything_if_outdated()
 
-post_social_login.connect(fetch_on_orcid_login)
+pre_social_login.connect(fetch_on_orcid_login)
```

Comment:
We need `pre_social_login` and not `post_social_login` here to prevent any account creation if we don't have a name.